plugins/apel/src/auditor_apel_plugin/core.py

```diff
@@ -16,16 +16,47 @@
 from cryptography import x509
 from cryptography.hazmat.primitives import hashes, serialization
 from cryptography.hazmat.primitives.serialization import pkcs7
+from pyauditor import Value, Operator, MetaOperator, MetaQuery, QueryBuilder
 
 
-def get_records(client, start_time, delay_time):
+def get_records(config, client, start_time, delay_time, site=None, end_time=None):
+    sites_to_report = json.loads(config["site"].get("sites_to_report"))
+    meta_key_site = config["auditor"].get("meta_key_site")
+
+    site_ids = []
+
+    if site is not None:
+        site_ids = sites_to_report[site]
+        logging.info(f"Getting records for site {site} with site_ids: {site_ids}")
+    else:
+        items = sites_to_report.items()
+        for k, v in items:
```

Comment:
Since you don't use items anywhere else, can't we just do `for k, v in sites_to_report.items():`?